Yuneec_SDK_iOS/YNCSDKConnection.h

```diff
@@ -9,6 +9,64 @@
 
 #import <Foundation/Foundation.h>
 
+/**
+ Data type for connection results.
+ */
+typedef NS_ENUM (NSInteger, YNCConnectionResult) {
```

Comment:
I think that should live in `YNCSDKConnection`.

Reply:
right!

Reply:
Actually I saw that the usual convention is to declare enums at the top of the file, outside @interface declaration. Since the scope does not change.

Reply:
Sorry, I think I meant it should be in Connection and not in Util.

Reply:
Okay. It is already defined in Connection. Did you mean this function getConnectionResultString() should be in Connection?

Reply:
Yer, I'd say so.

Reply:
okay. Committed the changes.